react/src/types/react-table-config.d.ts

```diff
@@ -0,0 +1,120 @@
+import {
```

Comment:
Maybe a comment with a "Why is this needed" and a link (SO, github issue to the react-table project etc) could help here?

Reply:
I added a comment :)